cli/assets/templates/go/src/w4/wasm4.go

```diff
@@ -47,34 +45,34 @@ const BUTTON_DOWN byte = 128;
 
 /** Copies pixels to the framebuffer. */
 //go:export blit
-func blit (sprite *byte, x int, y int, width uint, height uint, flags uint);
+func Blit(sprite *byte, x int, y int, width uint, height uint, flags uint)
 
 /** Copies a subregion within a larger sprite atlas to the framebuffer. */
 //go:export blitSub
-func blitSub (sprite *byte, x int, y int, width uint, height uint,
-    srcX uint, srcY uint, stride int, flags uint);
+func BlitSub(sprite *byte, x int, y int, width uint, height uint,
+	srcX uint, srcY uint, stride int, flags uint)
```

Comment:
Let's stick with 4 spaces indentation:

```suggestion
    srcX uint, srcY uint, stride int, flags uint)
```